de.mediathekview.MediathekView.yaml

```diff
@@ -0,0 +1,104 @@
+app-id: de.mediathekview.MediathekView
```

Comment:
Given it's strongly coupled with VLC, I wonder if it wouldn't be better to bundle VLC with the app. This can be easily achieved by using it as BaseApp.

Reply:
@barthalion You can perfectly use MediathekView without VLC, if you just use it to record live streams (this requires ffmpeg which is used as an extension) or download shows (this doesn't require anything at all).

I can make VLC a base app of Mediathek, but wouldn't this install VLC twice for everyone who also installs VLC explicitly?

Reply:
If it's usable without VLC then it's fine as it is.

Reply:
I think a base app would be surprising in so far as it'd duplicate VLC settings: If a user has customized their regular flatpak VLC these settings wouldn't propagate into the MediathekView sandbox, would they?

Reply:
It might be true, depending on whether VLC respects `$XDG_CONFIG_HOME` env var. But yeah, feel free to ignore it.

Reply:
I think it does. I understand flathub has the possibility of beta builds into a separate repository?  If so couldn't I use this to test a separate MediathekView manifest with VLC as base app?  

And then, if it works well, use that as new regular manifest?